src/NBitcoin/Block.cs

```diff
@@ -428,6 +429,23 @@ public static Block ParseJson(string json)
             return blk;
         }
 
+        public string ToJson(bool indented = false)
```

Comment:
I actually really think we should create a BlockModel that is only used by the controller.
The controller would then get the block and map it to a new blockModel object. This way there is no disruption to the Block object.
As for future proofing of Block vs SmartBlock serialization, we can always have custom serializers in place.